a11y/a11y.test.js

```diff
@@ -0,0 +1,67 @@
+import fs from 'fs'
+import path from 'path'
+
+import puppeteer from 'puppeteer'
+import execa from 'execa'
+
+import { mockServer } from './mock-server'
+
+const OTP_RR_CONFIG_FILE_PATH = './config.yml'
+const OTP_RR_CONFIG_BACKUP_PATH = './config.non-test.yml'
+const OTP_RR_TEST_CONFIG_PATH = './a11y/test-config.yml'
+
+beforeEach(() => {
+  // backup current config file
+  if (fs.existsSync(OTP_RR_CONFIG_FILE_PATH)) {
+    fs.renameSync(
+      OTP_RR_CONFIG_FILE_PATH,
+      OTP_RR_CONFIG_BACKUP_PATH
+    )
+    console.log('Backed up current OTP-RR config file')
+  }
+  // copy over test config file
+  fs.copyFileSync(
+    OTP_RR_TEST_CONFIG_PATH,
+    OTP_RR_CONFIG_FILE_PATH
+  )
+  console.log('Copied a11y test config file')
+
+  // Build OTP-RR main.js using new config file
+  execa.sync('yarn', ['build'])
+  console.log('Built OTP-RR')
+
+  // Launch mock OTP server
+  const MOCK_SERVER_PORT = 9999
```

Comment:
This port won't work with the ports allowed with our localhost Mapbox token.

Reply:
Leaflet fails a11y tests and can't really be fixed from within this repo (the copyright at the bottom) so it should be alright?